### ISSUE TITLE
Docs: the lazy components should add a note that they cannot work directly from server components.

### ISSUE THREAD

**Issue by razb-viola** [vercel/next.js#60656]
### What is the improvement or update you wish to see?

State that when you import client component from server component, lazy, it will not work.
You must wrap your client component with another client component and lazy load it from there.
Otherwise you will get inflated bundle size. 

### Is there any context that might help us understand?

I am trying to reduce the bundle size involving dynamic imports of client components.

**Comment by michael-harvey** [comment]
Are you able to provide an example, or the errors you're getting?

Seems like importing a client component into a page server component is possible.

**Comment by razb-viola** [comment]
There is no error. 
When you lazy import a component from a server component it will get into the bundle and not lazy imported.
You must load a client component that will lazy load that component instead - only then - the component won't get into the bundle and loaded as a separate chunk.

It either should be fixed or stated in the docs.

**Comment by michael-harvey** [comment]
Yeah I am able to replicate this issue. Definitely would be great to have some kind of explanation as to what is happening here from someone who understands this thoroughly 👍🏼

**Comment by razb-viola** [comment]
Great. Of course the better option is that will be allowed to lazy import client components from server components without the need to do an additional step of wrapping them.

**Comment by wesnolte** [comment]
I have the same issue, and others have logged similar issues here albeit with different impacts e.g. https://github.com/vercel/next.js/issues/61574
